webapp/src/css/inbox.less

```diff
@@ -1452,29 +1470,37 @@ mm-sidebar-menu .mat-sidenav-container {
         flex-direction: row;
         align-items: center;
         justify-content: center;
+    
         .button-label {
```

Comment:
The browser is ignoring this rule (see screenshot where attributes are scratched)

<img width="437" alt="Screenshot 2024-10-28 at 8 23 30 PM" src="https://github.com/user-attachments/assets/1c5deabc-fd0f-4a23-bed5-64472472893e">

And no cleaning the styles for mobile

<img width="435" alt="Screenshot 2024-10-28 at 8 19 50 PM" src="https://github.com/user-attachments/assets/a3f854fa-fb72-45b2-be6c-14ddfc598346">

I think it's fixed by just adding `a` before the class, like this:

```suggestion
        a .button-label {
```

Reply:
Maybe,  that is why i had issues with !important